src/controllers/audience/audienceController.ts

```diff
@@ -0,0 +1,69 @@
+import {Request, Response, Router} from 'express'
+import {AudienceFactory} from '../../learning-catalogue/model/factory/audienceFactory'
+import {LearningCatalogue} from '../../learning-catalogue'
+import {Audience} from '../../learning-catalogue/model/audience'
+import {Validator} from '../../learning-catalogue/validator/validator'
+
+export class AudienceController {
+	learningCatalogue: LearningCatalogue
+	audienceValidator: Validator<Audience>
+	audienceFactory: AudienceFactory
+	router: Router
+
+	constructor(
+		learningCatalogue: LearningCatalogue,
+		audienceValidator: Validator<Audience>,
+		audienceFactory: AudienceFactory
+	) {
+		this.learningCatalogue = learningCatalogue
+		this.audienceValidator = audienceValidator
+		this.audienceFactory = audienceFactory
+		this.router = Router()
+		this.setPathParametersMapping()
+		this.setRouterPaths()
+	}
+
+	private setPathParametersMapping() {
```

Comment:
MAINTAINABILITY - Can we please extract this to a service? I have seen the same piece of code on 3 different controllers now. Method name doesn't do what it says